### PR TITLE
Mobile menu animation is slow #312

### PR DESCRIPTION
**Pull request by alexPopNicolae**
changed animation  times for navigation container and hamburger menu

### CODE REVIEW

**Comment by alexnm** [comment]
@alexPopNicolae can you try ease-out and apply translation from -100% to 70px? (currently it's -200% and 69px)

File: app/components/nav/hamburgerMenu.js [hunk 1]
```diff
@@ -26,19 +26,19 @@ const HamburgerMenu = ({ isOpen, onClick }) => (
             width: 25px;
             height: 3px;
             position: static;
-            transition: all 900ms ease-in-out;
+            transition: all 400ms ease-in-out;
```
Comment (alexnm) [inline app/components/nav/hamburgerMenu.js]: try ease-out

File: app/components/nav/index.js [hunk 2]
```diff
@@ -145,7 +145,7 @@ class Nav extends Component {
                 display: block;
                 transform: translate(0, 69px); // Nav's height - the bottom-border
                 width: 100%;
-                transition: all 900ms ease-in;
+                transition: all 400ms ease-in-out;
```
Comment (alexnm) [inline app/components/nav/index.js]: try ease-out

File: app/components/nav/index.js [hunk 3]
```diff
@@ -145,7 +145,7 @@ class Nav extends Component {
                 display: block;
                 transform: translate(0, 69px); // Nav's height - the bottom-border
```
Comment (alexnm) [inline app/components/nav/index.js]: I would use 70px here so the white border is still visible

File: app/components/nav/index.js [hunk 4]
```diff
@@ -155,7 +155,7 @@ class Nav extends Component {
                 position: fixed;
                 left: 0;
                 transform: translate(0%, -200%);
```
Comment (alexnm) [inline app/components/nav/index.js]: original position could work with -100% so the delay is not visible when toggling

File: app/components/nav/index.js (hunk 2 above)
Comment (alexPopNicolae) [reply app/components/nav/index.js]: done

File: app/components/nav/index.js (hunk 3 above)
Comment (alexPopNicolae) [reply app/components/nav/index.js]: done

File: app/components/nav/index.js (hunk 4 above)
Comment (alexPopNicolae) [reply app/components/nav/index.js]: done

**Pull request merged**